src/functional/ffront/foast_pretty_printer.py

```diff
@@ -139,11 +142,13 @@ def visit_TernaryExpr(self, node: foast.TernaryExpr, **kwargs) -> str:
 
     Return = as_fmt("return {value}")
 
+    BlockStmt = as_mako("${'\\n'.join(stmts)}")
```

Comment:
Feel free to ignore, but I just realized that mako is not needed at all here.
```suggestion
    BlockStmt = as_fmt("{'\n'.join(stmts)}")
```

Reply:
Format strings sadly don't support newline characters inside braces: https://stackoverflow.com/questions/44780357/how-to-use-newline-n-in-f-string-to-format-output-in-python-3-6